### DISCUSSION TITLE
Do I have to run DIA-NN twice to get XICs?

### DISCUSSION THREAD

**Discussion by JunYao-mynamesofewletters** [vdemichev/DiaNN#879] [Q&A] [answered]
Hi all!
I have generated some XICs using the --vis command already. 
However, my workflow is : 
1. run DIA-NN; 
2. pick some proteins then the peptides from the report; 
3. go to command line and run DIA-NN again adding --vis 20, pep1, pep2...

Search twice, wait twice. Is it the right way?

**Comment by vdemichev** [comment, marked as answer]
Yes, need to run twice. Alternatively, use the 1.8.2 beta 27 (and later versions) with --xic command, it will export full chromatograms for everything, and save for each raw file in a separate .parquet file.

**Reply by JunYao-mynamesofewletters** [reply]
Thanks a lot Vadim! 

Is  ".parquet file" as easy to access as .xlsx file ?

**Reply by vdemichev** [reply]
In R or Python - yes. With R arrow or Python pyarrow package.

**Reply by JunYao-mynamesofewletters** [reply]
Again MANY thanks ! (It took me some time to find that version and the compression ratio is amazing for .parquet files)
